readme.md

```diff
@@ -245,7 +245,25 @@ yarn
 yarn build-all
 ```
 
-If changing audit output, you'll likely also need to have the protocol-buffer compiler installed. See the [official installation instructions](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation) or consult your [favorite package manager](https://formulae.brew.sh/formula/protobuf) for your OS.
+#### installing protobuf
+If changing audit output, you'll need to have the protocol-buffer/protobuf compiler installed. (v3.7.1 is known to be compatible). These are the steps that have worked well for us:
+
+```sh
+# Download https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protobuf-python-3.7.1.zip
+# unzip it
+# open a shell within that folder
+
+cd python
+python setup.py build
+python setup.py test
+(cd .. && autogen.sh && configure && make)
+(cd .. && sudo make install)
+python setup.py build --cpp_implementation
+sudo python setup.py install --cpp_implementation
+```
+
+Also, see the [official installation instructions](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation). Note that using a package manager to install will result in the latest version which **isn't** compatible with our output.
```

Comment:
would `brew install protobuf@3.7.1` work though?

Reply:
homebrew supports versions like that? i swear it was very anti- installing non-latest versions.

assuming it does, i think this would work. it's interesting that the homebrew formula installs the binary and then selectively [sets up the python bindings](https://github.com/Homebrew/homebrew-core/blob/372d31769c739852cccdefd76813364cc7260f2e/Formula/protobuf.rb#L52-L53), but ignores the 7 other language bindings... :)

Reply:
they definitely used to make it very difficult but they deprecated `homebrew/versions` and made the process easier. there are some restrictions on the versions they will support but they roughly do support that syntax now

https://docs.brew.sh/Versions#acceptable-versioned-formulae

Reply:
some more fun reading https://stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula/9832084#9832084 :)

Reply:
AHHHHHHHHHH this is great. :)